pydevices/acq400Devices/acq400_base.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2017, Massachusetts Institute of Technology All rights reserved.
+# Copyright (c) 2020, Massachusetts Institute of Technology All rights reserved.
```

Comment:
rename the module with a leading _ this will skip the whole file when parsing for Devices -> better performance (#2065 )